### PR TITLE
Default manage action

### PR DESCRIPTION
**Pull request by bachstatter**
- If you can borrow ->  show borrow
- If you 10% to liquidation ratio -> show repay
- Other wise -> show deposit

### CODE REVIEW

File: v3/ui/src/pages/Manage/ManageActions.tsx [hunk 1]
```diff
@@ -137,6 +142,28 @@ export const ManageAction = () => {
...
+    if (params.manageAction) return;
+    if (!liquidityPosition.data) return;
+    if (!collateralType) return;
+    const cRatio = liquidityPosition.data.cRatio.div(100);
+    const canBorrow = cRatio.gt(collateralType.issuanceRatioD18);
+    if (canBorrow) {
+      setQueryParam({ manageAction: 'borrow' });
+      return;
+    }
+    const cRatioIsCloseToLiqRatio = cRatio.mul(0.9).lt(collateralType.liquidationRatioD18);
```
Comment (jmzwar) [inline v3/ui/src/pages/Manage/ManageActions.tsx:157]: This 0.9 just hardcoded for now?

Comment (bachstatter) [reply v3/ui/src/pages/Manage/ManageActions.tsx:157]: Yea I just came up with something that sorta make sense.

Comment (jmzwar) [reply v3/ui/src/pages/Manage/ManageActions.tsx:157]: Ok - I personally would add a note here so we don't forget

Comment (bachstatter) [reply v3/ui/src/pages/Manage/ManageActions.tsx:157]: Hmm yea, happy to add what should it say?

Comment (bachstatter) [reply v3/ui/src/pages/Manage/ManageActions.tsx:157]: It will always be hardcoded to something

Comment (jmzwar) [reply v3/ui/src/pages/Manage/ManageActions.tsx:157]: I'd just add a note saying about saying 
NOTE: Current liquidation ratio is 0.9

Comment (bachstatter) [reply v3/ui/src/pages/Manage/ManageActions.tsx:157]: It's more like, "If user's c-ratio is within 10% of the configured liquidation ratio, set the repay action as initial state"

Comment (bachstatter) [reply v3/ui/src/pages/Manage/ManageActions.tsx:157]: > can keep such thing in `constants` probably. As it is easy to miss this "configuration" value in the component code

Do you have a nice name for that constants?  CRATIO_LIQUIDATION_RATIO_THRESHOLD_REPAY_UI? haha

Comment (bachstatter) [reply v3/ui/src/pages/Manage/ManageActions.tsx:157]: `c-ratio_threshold` exists already. It's the value that dictates how much of from the issuance ratio you can be and still claim

**Comment by bachstatter** [comment]
Merging now, feel free to add a comment and move to constants if you can figure out a way to not make it more confusing

**Pull request merged**